webpack.prod.config.babel.js

```diff
@@ -128,19 +75,26 @@ const settings = {
     alias: {
       'normalize.css': 'normalize.css/normalize.css',
     },
-    root: [
+    modules: [
       path.resolve(__dirname),
       path.resolve('./src'),
+      'node_modules',
     ],
-    modulesDirectories: ['node_modules'],
-    extensions: ['', '.js', '.jsx'],
+    extensions: ['.js', '.jsx'],
   },
 };
 
 if (config.get('enablePostCssLoader')) {
-  settings.postcss = [
-    autoprefixer({ browsers: ['last 2 versions'] }),
-  ];
+  settings.plugins.push(
+    new webpack.LoaderOptionsPlugin({
+      options: {
+        context: path.resolve(__dirname),
+        postcss: [
+          autoprefixer({ browsers: ['last 2 versions'] }),
```

Comment:
Looks like it's working. I searched `-ms-flex-pack` in `dist/amo-eb2fa8bf265a819a3276f84cff2ff9b2.css` before and after to verify.